README.md

```diff
@@ -62,6 +63,47 @@ To install, do:
 $ pip install snakepit
 ```
 
+## Use PyRun as Python distribution to build a spec-file
+
+You do not need any make-opt-writable rpm or svn2rpm. Simply build your spec-
+file in userspace and reduce the rpm size more than by half.
+
+### Special optional configuration for PyRun
+The PyRun binary version 2.1.1 we download was compiled against OpenSSL libraries
+which are different in minor version and naming from the one on server we wan't 
+to install the RPM on
+ 
+The ```libraries``` argument was introduced, so the spec-file can do some
+linkage magic for you.
+It is a dict with {'TARGET': 'LINK_NAME'}
+
+Example:
+```
+libraries: {'/usr/lib64/libssl.so':'libssl.so.1.0.0', '/usr/lib64/libcrypto.so':'libcrypto.so.1.0.0'}
```

Comment:
The cost of pushing this change into the next round is that then you have to deal with obsoleting the library stuff from already installed systems.

Another risk: By including links from /*/lib64/* you actually pin the package to be 64bit without hard-coding that in the spec file.

Either be platform-independant or hard-code your target platform everywhere.